frontend/src/pages/Interventions/ActivitiesComponent.tsx

```diff
@@ -19,6 +23,21 @@ const ActivityComponent: FunctionComponent<ActivityComponentProp> = ({
   activities,
   updateIntervention,
 }): ReactElement => {
+  const activity2ImageUrls = useMemo(() => {
```

Comment:
unrelated bug, was just getting annoyed cause this kept re-rendering and then it would sometimes show a fake error for a split second